### PR TITLE
Story/GAT-591 DAR-Pre populate

### PR DESCRIPTION
**Pull request by kandaj**

### CODE REVIEW

File: src/pages/DataAccessRequest/components/DropdownCustom/DropdownCustom.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
+import React from 'react';
```
Comment (ThinkingMonkeyLtd) [inline src/pages/DataAccessRequest/components/DropdownCustom/DropdownCustom.js:1]: Any way we can use the common dropdown for this?

File: test/DropdownCustom.test.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+afterEach(function () {
+	moxios.uninstall();
+	wrapper.unmount();  
+});
+
+describe('<DropdownCustom />', () => {
+
+    it('allows typing in the text input', async done => {
+        await moxios.wait(jest.fn);
```
Comment (ThinkingMonkeyLtd) [inline test/DropdownCustom.test.js:22]: It would be better to use mockMSW as with our services

File: test/mocks/dataMock.js [hunk 3]
```diff
@@ -5235,6 +5235,89 @@ const _activeTopicMultipleDatasets = {
 	},
 	datasets: [{ datasetId: '3814fc7e-02af-41ff-af00-4076ca325d5e', publisher: 'ALLIANCE > HQIP' }],
 };
+
+const _contributorsInfo = [
```
Comment (ThinkingMonkeyLtd) [inline test/mocks/dataMock.js:5239]: If this is the same as a response from an endpoint can we move it into the mock msw's in the service folder

**Review by ThinkingMonkeyLtd — APPROVED** [review]

**Review by dannitahdruk — APPROVED** [review]
lgtm!

**Pull request merged**